core/templates/pages/feedback-updates-page/feedback-updates-page-root.component.spec.ts

```diff
@@ -0,0 +1,159 @@
+    () => {
+      expect(component).toBeDefined();
+    });
+
+  it('should initialize and show page when access is valid', fakeAsync(() => {
+    spyOn(accessValidationBackendApiService, 'validateCanManageOwnAccount')
+      .and.returnValue(Promise.resolve());
+    spyOn(loaderService, 'showLoadingScreen');
+    spyOn(loaderService, 'hideLoadingScreen');
+
+    component.ngOnInit();
+    tick();
+
+    expect(loaderService.showLoadingScreen).toHaveBeenCalled();
+    expect(accessValidationBackendApiService.validateCanManageOwnAccount)
+      .toHaveBeenCalled();
+    expect(component.pageIsShown).toBeTrue();
+    expect(component.errorPageIsShown).toBeFalse();
+    expect(loaderService.hideLoadingScreen).toHaveBeenCalled();
+  }));
+
+  it('should initialize and show error page when server respond with error',
+    fakeAsync(() => {
+      spyOn(accessValidationBackendApiService, 'validateCanManageOwnAccount')
+        .and.returnValue(Promise.reject());
+      spyOn(loaderService, 'showLoadingScreen');
+      spyOn(loaderService, 'hideLoadingScreen');
+
+      component.ngOnInit();
+      tick();
+
+      expect(loaderService.showLoadingScreen).toHaveBeenCalled();
+      expect(accessValidationBackendApiService.validateCanManageOwnAccount)
+        .toHaveBeenCalled();
+      expect(component.pageIsShown).toBeFalse();
+      expect(component.errorPageIsShown).toBeTrue();
+      expect(loaderService.hideLoadingScreen).toHaveBeenCalled();
+    }));
+
+  it('should initialize and subscribe to onLangChange', fakeAsync(() => {
+    spyOn(accessValidationBackendApiService, 'validateCanManageOwnAccount')
+      .and.returnValue(Promise.resolve());
+    spyOn(component.directiveSubscriptions, 'add');
+    spyOn(translateService.onLangChange, 'subscribe');
+
+    component.ngOnInit();
+    tick();
+
+    expect(component.directiveSubscriptions.add).toHaveBeenCalled();
+    expect(translateService.onLangChange.subscribe).toHaveBeenCalled();
+  }));
+
+  it('should update page title whenever the language changes', () => {
+    spyOn(accessValidationBackendApiService, 'validateCanManageOwnAccount')
+      .and.returnValue(Promise.resolve());
+    component.ngOnInit();
+    spyOn(component, 'setPageTitleAndMetaTags');
+
+    translateService.onLangChange.emit();
+
+    expect(component.setPageTitleAndMetaTags).toHaveBeenCalled();
+  });
+
+  it('should obtain translated title and set the title and meta tags', () => {
+    spyOn(translateService, 'instant').and.callThrough();
+    spyOn(pageHeadService, 'updateTitleAndMetaTags');
+
+    component.setPageTitleAndMetaTags();
+
+    expect(translateService.instant).toHaveBeenCalledWith(
+      AppConstants.PAGES_REGISTERED_WITH_FRONTEND.FEEDBACK_UPDATES.TITLE);
+    expect(pageHeadService.updateTitleAndMetaTags).toHaveBeenCalledWith(
+      AppConstants.PAGES_REGISTERED_WITH_FRONTEND.FEEDBACK_UPDATES.TITLE,
+      AppConstants.PAGES_REGISTERED_WITH_FRONTEND.FEEDBACK_UPDATES.META);
+  });
```

Comment:
Is it possible to combine these 2 tests?

Reply:
Hi, @gp201 Thanks for the review,
These tests are already written separately in many files in codebase and I have just use them for me.
some of the files are here-->
Please see
1) https://github.com/oppia/oppia/blob/develop/core/templates/pages/preferences-page/preferences-page-root.component.spec.ts#L82

2)https://github.com/oppia/oppia/blob/develop/core/templates/pages/profile-page/profile-page-root.component.spec.ts#L132
3) https://github.com/oppia/oppia/blob/develop/core/templates/pages/delete-account-page/delete-account-page-root.component.spec.ts#L84